distribution-spec/v1.0/artifactory/README.md

```diff
@@ -0,0 +1,47 @@
+![Artifactory Logo](https://media.jfrog.com/wp-content/uploads/2022/05/24134044/artifactory-Icon.png)
+# jFrog Artifactory
+
+
+
+### Get Artifactory
+#### * Try out Artifactory - start [here](https://jfrog.com/start-free/)
+#### * Create a local OCI repository called oci-local
+
+```bash
+docker login <artifactory-url>
+```
+---
+
+## Run the conformance test
+
+---
+
+
+#### Clone OCI Distribution Spec
+
+```bash
+git clone https://github.com/opencontainers/distribution-spec
+```
```

Comment:
since this ifs for v1.0, please run against the v1.0.1 tag of the spec:

```
git clone https://github.com/opencontainers/distribution-spec -b v1.0.1
```

In report.html, the "Test Version" should be equal to 10efb871ee08eb13678fe275e2cf89b3df86542b (v1.0.1)

You could run inside the following container image:
```
ghcr.io/opencontainers/distribution-spec/conformance:v1.0.1
```

Reply:
https://github.com/orgs/opencontainers/packages/container/distribution-spec%2Fconformance/10447210?tag=v1.0.1

Reply:
I ran it inside the container image and updated the results